plugins/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "appium-inspector-plugin",
-  "version": "2024.12.1-beta.10",
+  "version": "2024.12.1-beta.12",
```

Comment:
should the version be bumped automatically?

Reply:
Yes, when we do `npm version <version>` in the top to release a new electron edition.

This is to match with the current latest `npm` package version. I wrongly published `2024.12.1` before for this plugin... so currently here keeps beta. The next release will be non-beta by syncing electron's one, thus this PR itself ok with this.